pkg/volume/plugins.go

```diff
@@ -121,6 +122,9 @@ type VolumeHost interface {
 	// the provided spec.  See comments on NewWrapperBuilder for more
 	// context.
 	NewWrapperCleaner(spec *Spec, podUID types.UID, mounter mount.Interface) (Cleaner, error)
+
+	//Get cloud provider from kubelet
+	GetCloudProvider() cloudprovider.Interface
```

Comment:
The `cloudprovider` package [already provides](https://github.com/kubernetes/kubernetes/blob/master/pkg/cloudprovider/plugins.go#L55) a `GetCloudProvider(...)` method that should be used instead.